bedrock/firefox/templates/firefox/new/batm/scene2.html

```diff
@@ -0,0 +1,34 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/batm/base.html" %}
+
+{% block body_class %}scene-2{% endblock %}
+
+{% block string_data %}
+  data-pixels="{% for pixel in settings.TRACKING_PIXELS %}{{ pixel }}{% if not loop.last %}::{% endif %}{% endfor %}"
```

Comment:
Oh, yes we should add it there too, else they can't measure the ad spend. I can do it in this PR.